src/contracts/permissions/PermissionController.sol

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.27;
+
+import "@openzeppelin-upgrades/contracts/proxy/utils/Initializable.sol";
+import "./PermissionControllerStorage.sol";
+
+contract PermissionController is Initializable, PermissionControllerStorage {
+    using EnumerableSet for *;
+
+    modifier onlyAdmin(
+        address account
+    ) {
+        require(isAdmin(account, msg.sender), NotAdmin());
+        _;
+    }
+
+    /**
+     *
+     *                         INITIALIZING FUNCTIONS
+     *
+     */
+    constructor() {
+        _disableInitializers();
+    }
+
+    function initialize() external initializer {}
+
+    /**
+     *
+     *                         EXTERNAL FUNCTIONS
+     *
+     */
+
+    /// @inheritdoc IPermissionController
+    function setAdmin(address account, address admin) external onlyAdmin(account) {
```

Comment:
addAdmin

isAdmin(account)

Reply:
Changed function to addAdmin. Kept onlyAdmin modifier given we have a function with same name: https://github.com/Layr-Labs/eigenlayer-contracts/pull/870/commits/ccb8c89bf32175dc20f9d2b77f179db465ab9f98